### DISCUSSION TITLE
How to only do CSS styling for one slide?

### DISCUSSION THREAD

**Discussion by wietyy** [marp-team/marp#440] [Q&A] [answered]
Hey! I just started using Marp. I know that CSS styling works, but I want to know how to only apply styling to one slide.

For example:

```markdown
# Slide 1
foo

---
<style>
h1 {
     color: black
}
</style>
# Slide 2
bar
```

I would like the `color: black` to **only** apply to **Slide 2**. Whenever I do this, however, it affects the # on slide 1 too.

Any response would be well appreciated!

**Comment by yhatt** [comment, marked as answer]
`<style>` tag is working same as HTML, setting styles _for the whole of the document_.

Try using `<style scoped>` instead. Marp provides the support of `scoped` attribute for `<style>` to limit the effect of styles to the current slide.
https://marpit.marp.app/theme-css?id=scoped-style.

**Reply by wietyy** [reply]
Thanks a lot!
